[core-hw-kit] app-pda/libimobiledevice-1.3.0_p20240330 fails to merge

```
/var/tmp/portage/app-pda/libimobiledevice-1.3.0_p20240330/work/libimobiledevice-1.3.0_p20240330/cython/installation_proxy.pxi:293:77: Cannot assign type 'void (plist_t, plist_t, void *) except * nogil' to 'instproxy_status_cb_t' (alias of 'void (*)(plist_t, plist_t, void *) noexcept'). Exception values are incompatible. Suggest adding 'noexcept' to the type of 'instproxy_notify_cb'.
make: *** [Makefile:806: imobiledevice.c] Error 1
make: Leaving directory '/var/tmp/portage/app-pda/libimobiledevice-1.3.0_p20240330/work/libimobiledevice-1.3.0_p20240330_build-python3_9/cython'
 * ERROR: app-pda/libimobiledevice-1.3.0_p20240330::core-hw-kit failed (compile phase):
 *   emake failed
```

--- a/core-hw-kit/curated/app-pda/libimobiledevice/autogen.yaml
+++ b/core-hw-kit/curated/app-pda/libimobiledevice/autogen.yaml
@@ -2,9 +2,9 @@ libimobiledevice:
   generator: github-1
   packages:
     - libimobiledevice:
-        version: 1.3.0_p20240330
+        version: 1.3.0_p20240401
         github:
           user: libimobiledevice
           repo: libimobiledevice
           query: snapshot
-          snapshot: 9649448434ab5c674d2cc11f76e69e6ee5e9dc09
+          snapshot: 1ec2c2c5e3609cc02b302bcbd79ed2872260d350

